src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -42,7 +46,7 @@ internal HttpInMetricsListener(string name, Meter meter, AspNetCoreMetricsInstru
     {
         this.meter = meter;
         this.options = options;
-        this.httpServerDuration = meter.CreateHistogram<double>(HttpServerDurationMetricName, "ms", "Measures the duration of inbound HTTP requests.");
+        this.httpServerDuration = meter.CreateHistogram<double>(HttpServerDurationMetricName, "s", "Measures the duration of inbound HTTP requests.");
```

Comment:
[the ](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs#L147) must be changed, right? Its still doing ms, not seconds..

Reply:
Thanks for catching that. I was focused on figuring out the View and the unit test working that I missed this.